Tasks/UseDotNetV1/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -0,0 +1,31 @@
+{
+  "loc.friendlyName": ".NET Core-Toolinstaller",
```

Comment:
fyi, loc build will revert these to English until the translations come back from the localizers

Reply:
to head of future confusion when these strings are reverted to english, you might want to delete these files and let the build generate the english ones instead.